tests/UI/campaigns/functional/BO/14_advancedParameters/08_logs/01_filterSortAndPagination.ts

```diff
@@ -93,14 +88,14 @@ describe('BO - Advanced Parameters - Logs : Filter, sort and pagination logs tab
 
       await dashboardPage.goToSubMenu(page, dashboardPage.advancedParametersLink, dashboardPage.logsLink);
 
-      const pageTitle = await logsPage.getPageTitle(page);
+      const pageTitle: string = await logsPage.getPageTitle(page);
```

Comment:
as `logsPage.getPageTitle(page)` return should be typed as a string, this `string` type looks unnecessary

Reply:
Fixed